Food Mixin Inject Crush

```log
ElyPrismLauncher version: 9.4 (official)


Launched instance in online mode

account.ely.by resolves to:
    [**.**.**.**]


ely.by resolves to:
    [**.**.**.**]


Minecraft folder is:
C:/Users/********/AppData/Roaming/ElyPrismLauncher/instances/Test/minecraft


Java path is:
C:/Users/********/AppData/Roaming/ElyPrismLauncher/java/java-runtime-gamma/bin/javaw.exe


Java is version 17.0.8, using 64 (amd64) architecture, from Microsoft.


Main Class:
  io.github.zekerzhayard.forgewrapper.installer.Main

Native path:
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/instances/Test/natives

Traits:
traits XR:Initial
traits FirstThreadOnMacOS
traits feature:is_quick_play_multiplayer
traits feature:is_quick_play_singleplayer

Libraries:
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-glfw-natives-windows-arm64/3.3.1/lwjgl-glfw-natives-windows-arm64-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-glfw-natives-windows-x86/3.3.1/lwjgl-glfw-natives-windows-x86-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-glfw-natives-windows/3.3.1/lwjgl-glfw-natives-windows-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-glfw/3.3.1/lwjgl-glfw-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-jemalloc-natives-windows-arm64/3.3.1/lwjgl-jemalloc-natives-windows-arm64-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-jemalloc-natives-windows-x86/3.3.1/lwjgl-jemalloc-natives-windows-x86-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-jemalloc-natives-windows/3.3.1/lwjgl-jemalloc-natives-windows-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-jemalloc/3.3.1/lwjgl-jemalloc-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-natives-windows-arm64/3.3.1/lwjgl-natives-windows-arm64-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-natives-windows-x86/3.3.1/lwjgl-natives-windows-x86-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-natives-windows/3.3.1/lwjgl-natives-windows-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-openal-natives-windows-arm64/3.3.1/lwjgl-openal-natives-windows-arm64-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-openal-natives-windows-x86/3.3.1/lwjgl-openal-natives-windows-x86-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-openal-natives-windows/3.3.1/lwjgl-openal-natives-windows-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-openal/3.3.1/lwjgl-openal-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-opengl-natives-windows-arm64/3.3.1/lwjgl-opengl-natives-windows-arm64-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-opengl-natives-windows-x86/3.3.1/lwjgl-opengl-natives-windows-x86-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-opengl-natives-windows/3.3.1/lwjgl-opengl-natives-windows-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-opengl/3.3.1/lwjgl-opengl-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-stb-natives-windows-arm64/3.3.1/lwjgl-stb-natives-windows-arm64-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-stb-natives-windows-x86/3.3.1/lwjgl-stb-natives-windows-x86-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-stb-natives-windows/3.3.1/lwjgl-stb-natives-windows-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-stb/3.3.1/lwjgl-stb-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-tinyfd-natives-windows-arm64/3.3.1/lwjgl-tinyfd-natives-windows-arm64-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-tinyfd-natives-windows-x86/3.3.1/lwjgl-tinyfd-natives-windows-x86-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-tinyfd-natives-windows/3.3.1/lwjgl-tinyfd-natives-windows-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl-tinyfd/3.3.1/lwjgl-tinyfd-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/lwjgl/lwjgl/3.3.1/lwjgl-3.3.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/com/github/oshi/oshi-core/6.2.2/oshi-core-6.2.2.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/com/google/code/gson/gson/2.10/gson-2.10.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/com/google/guava/failureaccess/1.0.1/failureaccess-1.0.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/com/google/guava/guava/31.1-jre/guava-31.1-jre.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/com/ibm/icu/icu4j/71.1/icu4j-71.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/com/mojang/authlib/4.0.43/authlib-4.0.43.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/com/mojang/blocklist/1.0.10/blocklist-1.0.10.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/com/mojang/brigadier/1.1.8/brigadier-1.1.8.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/com/mojang/datafixerupper/6.0.8/datafixerupper-6.0.8.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/com/mojang/logging/1.1.1/logging-1.1.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/com/mojang/patchy/2.2.10/patchy-2.2.10.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/com/mojang/text2speech/1.17.9/text2speech-1.17.9.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/commons-codec/commons-codec/1.15/commons-codec-1.15.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/commons-io/commons-io/2.11.0/commons-io-2.11.0.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/commons-logging/commons-logging/1.2/commons-logging-1.2.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/io/netty/netty-buffer/4.1.82.Final/netty-buffer-4.1.82.Final.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/io/netty/netty-codec/4.1.82.Final/netty-codec-4.1.82.Final.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/io/netty/netty-common/4.1.82.Final/netty-common-4.1.82.Final.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/io/netty/netty-handler/4.1.82.Final/netty-handler-4.1.82.Final.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/io/netty/netty-resolver/4.1.82.Final/netty-resolver-4.1.82.Final.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/io/netty/netty-transport-classes-epoll/4.1.82.Final/netty-transport-classes-epoll-4.1.82.Final.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/io/netty/netty-transport-native-unix-common/4.1.82.Final/netty-transport-native-unix-common-4.1.82.Final.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/io/netty/netty-transport/4.1.82.Final/netty-transport-4.1.82.Final.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/it/unimi/dsi/fastutil/8.5.9/fastutil-8.5.9.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/net/java/dev/jna/jna-platform/5.12.1/jna-platform-5.12.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/net/java/dev/jna/jna/5.12.1/jna-5.12.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/net/sf/jopt-simple/jopt-simple/5.0.4/jopt-simple-5.0.4.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/apache/commons/commons-compress/1.21/commons-compress-1.21.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/apache/commons/commons-lang3/3.12.0/commons-lang3-3.12.0.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/apache/httpcomponents/httpclient/4.5.13/httpclient-4.5.13.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/apache/httpcomponents/httpcore/4.4.15/httpcore-4.4.15.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/apache/logging/log4j/log4j-api/2.19.0/log4j-api-2.19.0.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/apache/logging/log4j/log4j-core/2.19.0/log4j-core-2.19.0.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/apache/logging/log4j/log4j-slf4j2-impl/2.19.0/log4j-slf4j2-impl-2.19.0.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/joml/joml/1.10.5/joml-1.10.5.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/slf4j/slf4j-api/2.0.1/slf4j-api-2.0.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/io/github/zekerzhayard/ForgeWrapper/prism-2024-02-29/ForgeWrapper-prism-2024-02-29.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/cpw/mods/securejarhandler/2.1.10/securejarhandler-2.1.10.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/ow2/asm/asm/9.7.1/asm-9.7.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/ow2/asm/asm-commons/9.7.1/asm-commons-9.7.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/ow2/asm/asm-tree/9.7.1/asm-tree-9.7.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/ow2/asm/asm-util/9.7.1/asm-util-9.7.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/ow2/asm/asm-analysis/9.7.1/asm-analysis-9.7.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/net/minecraftforge/accesstransformers/8.0.4/accesstransformers-8.0.4.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/antlr/antlr4-runtime/4.9.1/antlr4-runtime-4.9.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/net/minecraftforge/eventbus/6.0.5/eventbus-6.0.5.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/net/minecraftforge/forgespi/7.0.1/forgespi-7.0.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/net/minecraftforge/coremods/5.2.1/coremods-5.2.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/cpw/mods/modlauncher/10.0.9/modlauncher-10.0.9.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/net/minecraftforge/unsafe/0.2.0/unsafe-0.2.0.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/net/minecraftforge/mergetool/1.1.5/mergetool-1.1.5-api.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/com/electronwill/night-config/core/3.6.4/core-3.6.4.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/com/electronwill/night-config/toml/3.6.4/toml-3.6.4.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/apache/maven/maven-artifact/3.8.5/maven-artifact-3.8.5.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/net/jodah/typetools/0.6.3/typetools-0.6.3.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/net/minecrell/terminalconsoleappender/1.2.0/terminalconsoleappender-1.2.0.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/jline/jline-reader/3.12.1/jline-reader-3.12.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/jline/jline-terminal/3.12.1/jline-terminal-3.12.1.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/spongepowered/mixin/0.8.5/mixin-0.8.5.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/openjdk/nashorn/nashorn-core/15.4/nashorn-core-15.4.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/net/minecraftforge/JarJarSelector/0.3.19/JarJarSelector-0.3.19.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/net/minecraftforge/JarJarMetadata/0.3.19/JarJarMetadata-0.3.19.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/cpw/mods/bootstraplauncher/1.1.2/bootstraplauncher-1.1.2.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/net/minecraftforge/JarJarFileSystems/0.3.19/JarJarFileSystems-0.3.19.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/net/minecraftforge/fmlloader/1.20.1-47.3.12/fmlloader-1.20.1-47.3.12.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/net/minecraftforge/fmlearlydisplay/1.20.1-47.3.12/fmlearlydisplay-1.20.1-47.3.12.jar
  C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/com/mojang/minecraft/1.20.1/minecraft-1.20.1-client.jar

Native libraries:

Mods:
  [✔] geckolib-forge-1.20.1-4.7.1.3
  [✔] the-point-of-no-return-7116e0e

Params:
  --username  --version 1.20.1 --gameDir C:/Users/********/AppData/Roaming/ElyPrismLauncher/instances/Test/minecraft --assetsDir C:/Users/********/AppData/Roaming/ElyPrismLauncher/assets --assetIndex 5 --uuid  --accessToken  --userType  --versionType release --launchTarget forgeclient --fml.forgeVersion 47.3.12 --fml.mcVersion 1.20.1 --fml.forgeGroup net.minecraftforge --fml.mcpVersion 20230612.114412

Window size: 854 x 480

Launcher: standard

Java Arguments:
[-XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump, -Xms512m, -Xmx9088m, -Duser.language=en]


Minecraft process ID: 10728


Checking: MC_SLIM
Checking: MERGED_MAPPINGS
Checking: MAPPINGS
Checking: MC_EXTRA
Checking: MOJMAPS
Checking: PATCHED
Checking: MC_SRG
2025-06-17 16:35:48,646 main WARN Advanced terminal features are not available in this environment
[16:35:48] [main/INFO] [cp.mo.mo.Launcher/MODLAUNCHER]: ModLauncher running: args [--username, Jhon1212, --version, 1.20.1, --gameDir, C:/Users/********/AppData/Roaming/ElyPrismLauncher/instances/Test/minecraft, --assetsDir, C:/Users/********/AppData/Roaming/ElyPrismLauncher/assets, --assetIndex, 5, --uuid, <ID ПРОФИЛЯ>, --accessToken, ????????, --userType, msa, --versionType, release, --launchTarget, forgeclient, --fml.forgeVersion, 47.3.12, --fml.mcVersion, 1.20.1, --fml.forgeGroup, net.minecraftforge, --fml.mcpVersion, 20230612.114412, --width, 854, --height, 480]
[16:35:48] [main/INFO] [cp.mo.mo.Launcher/MODLAUNCHER]: ModLauncher 10.0.9+10.0.9+main.dcd20f30 starting: java version 17.0.8 by Microsoft; OS Windows 11 arch amd64 version 10.0
[16:35:49] [main/INFO] [ne.mi.fm.lo.ImmediateWindowHandler/]: Loading ImmediateWindowProvider fmlearlywindow
[16:35:49] [main/INFO] [EARLYDISPLAY/]: Trying GL version 4.6
[16:35:50] [main/INFO] [EARLYDISPLAY/]: Requested GL version 4.6 got version 4.6
[16:35:50] [main/INFO] [mixin/]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=union:/C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/org/spongepowered/mixin/0.8.5/mixin-0.8.5.jar%23150!/ Service=ModLauncher Env=CLIENT
[16:35:50] [pool-2-thread-1/INFO] [EARLYDISPLAY/]: GL info: Intel(R) Iris(R) Xe Graphics GL version 4.6.0 - Build 32.0.101.5542, Intel
[16:35:50] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file C:\Users\********\AppData\Roaming\ElyPrismLauncher\libraries\net\minecraftforge\fmlcore\1.20.1-47.3.12\fmlcore-1.20.1-47.3.12.jar is missing mods.toml file
[16:35:50] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file C:\Users\********\AppData\Roaming\ElyPrismLauncher\libraries\net\minecraftforge\javafmllanguage\1.20.1-47.3.12\javafmllanguage-1.20.1-47.3.12.jar is missing mods.toml file
[16:35:50] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file C:\Users\********\AppData\Roaming\ElyPrismLauncher\libraries\net\minecraftforge\lowcodelanguage\1.20.1-47.3.12\lowcodelanguage-1.20.1-47.3.12.jar is missing mods.toml file
[16:35:50] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file C:\Users\********\AppData\Roaming\ElyPrismLauncher\libraries\net\minecraftforge\mclanguage\1.20.1-47.3.12\mclanguage-1.20.1-47.3.12.jar is missing mods.toml file
[16:35:50] [main/INFO] [ne.mi.fm.lo.mo.JarInJarDependencyLocator/]: Found 1 dependencies adding them to mods collection
[16:35:52] [main/INFO] [mixin/]: Compatibility level set to JAVA_17
[16:35:52] [main/ERROR] [mixin/]: Mixin config mixin.ponr.json does not specify "minVersion" property
[16:35:52] [main/INFO] [cp.mo.mo.LaunchServiceHandler/MODLAUNCHER]: Launching target 'forgeclient' with arguments [--version, 1.20.1, --gameDir, C:\Users\********\AppData\Roaming\ElyPrismLauncher\instances\Test\minecraft, --assetsDir, C:\Users\********\AppData\Roaming\ElyPrismLauncher\assets, --uuid, <ID ПРОФИЛЯ>, --username, Jhon1212, --assetIndex, 5, --accessToken, ????????, --userType, msa, --versionType, release, --width, 854, --height, 480]
[16:35:57] [Datafixer Bootstrap/INFO] [mojang/DataFixerBuilder]: 188 Datafixer optimizations took 210 milliseconds
[16:35:59] [Render thread/WARN] [minecraft/VanillaPackResourcesBuilder]: Assets URL 'union:/C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/net/minecraft/client/1.20.1-20230612.114412/client-1.20.1-20230612.114412-srg.jar%23160!/assets/.mcassetsroot' uses unexpected schema
[16:35:59] [Render thread/WARN] [minecraft/VanillaPackResourcesBuilder]: Assets URL 'union:/C:/Users/********/AppData/Roaming/ElyPrismLauncher/libraries/net/minecraft/client/1.20.1-20230612.114412/client-1.20.1-20230612.114412-srg.jar%23160!/data/.mcassetsroot' uses unexpected schema
[16:35:59] [Render thread/INFO] [mojang/YggdrasilAuthenticationService]: Environment: authHost='https://authserver.ely.by/auth', accountsHost='https://account.ely.by/api/mojang', sessionHost='https://account.ely.by/api/authlib-injector/sessionserver', servicesHost='https://account.ely.by/api/mojang/services', name='ELY'
[16:35:59] [Render thread/INFO] [minecraft/Minecraft]: Setting user: Jhon1212
[16:35:59] [Render thread/INFO] [minecraft/Minecraft]: Backend library: LWJGL version 3.3.1 build 7
[16:35:59] [modloading-worker-0/INFO] [ne.mi.co.ForgeMod/FORGEMOD]: Forge mod loading, version 47.3.12, for MC 1.20.1 with MCP 20230612.114412
[16:35:59] [modloading-worker-0/INFO] [ne.mi.co.MinecraftForge/FORGE]: MinecraftForge v47.3.12 Initialized
[16:36:01] [Render thread/INFO] [minecraft/ReloadableResourceManager]: Reloading ResourceManager: vanilla, mod_resources
[16:36:01] [Worker-Main-5/INFO] [or.st.tp.TPONR/]: The Point Of No Return is starting
[16:36:01] [Worker-Main-2/INFO] [minecraft/UnihexProvider]: Found unifont_all_no_pua-15.0.06.hex, loading
[16:36:01] [Forge Version Check/INFO] [ne.mi.fm.VersionChecker/]: [forge] Starting version check at https://files.minecraftforge.net/net/minecraftforge/forge/promotions_slim.json
[16:36:01] [Worker-Main-6/WARN] [minecraft/SpriteLoader]: Texture tponr:block/clock with size 64x93 limits mip level from 4 to 0
[16:36:02] [Forge Version Check/INFO] [ne.mi.fm.VersionChecker/]: [forge] Found status: OUTDATED Current: 47.3.12 Target: 47.4.0
[16:36:02] [Worker-Main-10/WARN] [minecraft/ModelBakery]: Exception loading blockstate definition: 'tponr:blockstates/bunker_door_sub_block.json' missing model for variant: 'tponr:bunker_door_sub_block#facing=west'
[16:36:02] [Worker-Main-10/WARN] [minecraft/ModelBakery]: Exception loading blockstate definition: 'tponr:blockstates/bunker_door_sub_block.json' missing model for variant: 'tponr:bunker_door_sub_block#facing=east'
[16:36:02] [Worker-Main-10/WARN] [minecraft/ModelBakery]: Exception loading blockstate definition: 'tponr:blockstates/bunker_door_sub_block.json' missing model for variant: 'tponr:bunker_door_sub_block#facing=south'
[16:36:02] [Worker-Main-10/WARN] [minecraft/ModelBakery]: Exception loading blockstate definition: 'tponr:blockstates/bunker_door_sub_block.json' missing model for variant: 'tponr:bunker_door_sub_block#facing=north'
[16:36:04] [Render thread/WARN] [minecraft/SoundEngine]: Missing sound for event: minecraft:item.goat_horn.play
[16:36:04] [Render thread/WARN] [minecraft/SoundEngine]: Missing sound for event: minecraft:entity.goat.screaming.horn_break
[16:36:04] [Render thread/INFO] [mojang/Library]: OpenAL initialized on device OpenAL Soft on Динамики (Realtek High Definition Audio(SST))
[16:36:04] [Render thread/INFO] [minecraft/SoundEngine]: Sound engine started
[16:36:04] [Render thread/INFO] [minecraft/TextureAtlas]: Created: 1024x1024x4 minecraft:textures/atlas/blocks.png-atlas
[16:36:05] [Render thread/INFO] [minecraft/TextureAtlas]: Created: 256x256x4 minecraft:textures/atlas/signs.png-atlas
[16:36:05] [Render thread/INFO] [minecraft/TextureAtlas]: Created: 512x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[16:36:05] [Render thread/INFO] [minecraft/TextureAtlas]: Created: 512x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[16:36:05] [Render thread/INFO] [minecraft/TextureAtlas]: Created: 1024x1024x4 minecraft:textures/atlas/armor_trims.png-atlas
[16:36:05] [Render thread/INFO] [minecraft/TextureAtlas]: Created: 128x64x4 minecraft:textures/atlas/decorated_pot.png-atlas
[16:36:05] [Render thread/INFO] [minecraft/TextureAtlas]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[16:36:05] [Render thread/INFO] [minecraft/TextureAtlas]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[16:36:05] [Render thread/INFO] [minecraft/TextureAtlas]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[16:36:05] [Render thread/WARN] [minecraft/ShaderInstance]: Shader rendertype_entity_translucent_emissive could not find sampler named Sampler2 in the specified shader program.
[16:36:05] [Render thread/INFO] [minecraft/TextureAtlas]: Created: 256x256x0 minecraft:textures/atlas/particles.png-atlas
[16:36:05] [Render thread/INFO] [minecraft/TextureAtlas]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[16:36:05] [Render thread/INFO] [minecraft/TextureAtlas]: Created: 128x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[16:36:06] [Realms Notification Availability checker #1/INFO] [mojang/RealmsClient]: Could not authorize you against Realms server: java.lang.NullPointerException: Cannot invoke "java.util.Date.getTime()" because the return value of "com.nimbusds.jwt.JWTClaimsSet.getNotBeforeTime()" is null
[16:36:16] [Render thread/INFO] [minecraft/RecipeManager]: Loaded 7 recipes
[16:36:16] [Render thread/INFO] [minecraft/AdvancementList]: Loaded 1271 advancements
[16:36:17] [Server thread/INFO] [minecraft/IntegratedServer]: Starting integrated minecraft server version 1.20.1
[16:36:17] [Server thread/INFO] [minecraft/MinecraftServer]: Generating keypair
[16:36:18] [Server thread/INFO] [minecraft/MinecraftServer]: Preparing start region for dimension minecraft:overworld
[16:36:22] [Render thread/INFO] [minecraft/LoggerChunkProgressListener]: Preparing spawn area: 0%
[16:36:22] [Render thread/INFO] [minecraft/LoggerChunkProgressListener]: Preparing spawn area: 0%
[16:36:22] [Render thread/INFO] [minecraft/LoggerChunkProgressListener]: Preparing spawn area: 0%
[16:36:22] [Render thread/INFO] [minecraft/LoggerChunkProgressListener]: Preparing spawn area: 0%
[16:36:22] [Render thread/INFO] [minecraft/LoggerChunkProgressListener]: Preparing spawn area: 0%
[16:36:22] [Render thread/INFO] [minecraft/LoggerChunkProgressListener]: Preparing spawn area: 0%
[16:36:22] [Render thread/INFO] [minecraft/LoggerChunkProgressListener]: Preparing spawn area: 0%
[16:36:22] [Render thread/INFO] [minecraft/LoggerChunkProgressListener]: Preparing spawn area: 2%
[16:36:22] [Render thread/INFO] [minecraft/LoggerChunkProgressListener]: Preparing spawn area: 46%
[16:36:23] [Server thread/INFO] [ne.mi.se.pe.PermissionAPI/]: Successfully initialized permission handler forge:default_handler
[16:36:23] [Render thread/INFO] [minecraft/LoggerChunkProgressListener]: Time elapsed: 4410 ms
[16:36:23] [Server thread/INFO] [minecraft/IntegratedServer]: Changing view distance to 12, from 10
[16:36:23] [Server thread/INFO] [minecraft/IntegratedServer]: Changing simulation distance to 12, from 0
[16:36:23] [Render thread/WARN] [io.ne.ut.in.SystemPropertyUtil/]: Unable to parse the boolean system property 'java.net.preferIPv6Addresses':system - using the default value: false
[16:36:24] [Netty Local Client IO #0/INFO] [ne.mi.ne.NetworkHooks/]: Connected to a modded server.
[16:36:24] [Server thread/FATAL] [mixin/]: Mixin apply failed mixin.ponr.json:FoodDataMixin -> net.minecraft.world.food.FoodData: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException Critical injection failure: @ModifyConstant annotation on modifyEatMaxFoodLevel could not find any targets matching 'eat(IF)V' in net.minecraft.world.food.FoodData. No refMap loaded. [PREINJECT Applicator Phase -> mixin.ponr.json:FoodDataMixin -> Prepare Injections ->  -> constant$zza000$modifyEatMaxFoodLevel(I)I -> Parse]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Critical injection failure: @ModifyConstant annotation on modifyEatMaxFoodLevel could not find any targets matching 'eat(IF)V' in net.minecraft.world.food.FoodData. No refMap loaded. [PREINJECT Applicator Phase -> mixin.ponr.json:FoodDataMixin -> Prepare Injections ->  -> constant$zza000$modifyEatMaxFoodLevel(I)I -> Parse]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.validateTargets(InjectionInfo.java:656) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.findTargets(InjectionInfo.java:587) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.readAnnotation(InjectionInfo.java:330) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.<init>(InjectionInfo.java:316) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.injection.struct.ModifyConstantInjectionInfo.<init>(ModifyConstantInjectionInfo.java:54) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?] {}
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?] {}
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?] {}
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo$InjectorEntry.create(InjectionInfo.java:149) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.parse(InjectionInfo.java:708) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.prepareInjections(MixinTargetContext.java:1311) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.prepareInjections(MixinApplicatorStandard.java:1042) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:393) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:325) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:383) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:365) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:250) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.service.modlauncher.MixinTransformationHandler.processClass(MixinTransformationHandler.java:131) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.launch.MixinLaunchPluginLegacy.processClass(MixinLaunchPluginLegacy.java:131) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at cpw.mods.modlauncher.serviceapi.ILaunchPluginService.processClassWithFlags(ILaunchPluginService.java:156) ~[modlauncher-10.0.9.jar:10.0.9+10.0.9+main.dcd20f30] {}
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113) ~[securejarhandler-2.1.10.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219) ~[securejarhandler-2.1.10.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229) ~[securejarhandler-2.1.10.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219) ~[securejarhandler-2.1.10.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135) ~[securejarhandler-2.1.10.jar:?] {}
	at java.lang.ClassLoader.loadClass(ClassLoader.java:525) ~[?:?] {}
	at net.minecraft.world.entity.player.Player.<init>(Player.java:144) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.level.ServerPlayer.<init>(ServerPlayer.java:249) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.players.PlayerList.m_215624_(PlayerList.java:422) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading}
	at net.minecraft.server.network.ServerLoginPacketListenerImpl.m_10055_(ServerLoginPacketListenerImpl.java:121) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.network.ServerLoginPacketListenerImpl.m_9933_(ServerLoginPacketListenerImpl.java:70) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.network.Connection.m_129483_(Connection.java:263) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading}
	at net.minecraft.server.network.ServerConnectionListener.m_9721_(ServerConnectionListener.java:142) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading}
	at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:907) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:814) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.client.server.IntegratedServer.m_5705_(IntegratedServer.java:89) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:661) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:251) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at java.lang.Thread.run(Thread.java:833) ~[?:?] {}
[16:36:24] [Server thread/ERROR] [minecraft/MinecraftServer]: Encountered an unexpected exception
org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:250) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.service.modlauncher.MixinTransformationHandler.processClass(MixinTransformationHandler.java:131) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.launch.MixinLaunchPluginLegacy.processClass(MixinLaunchPluginLegacy.java:131) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at cpw.mods.modlauncher.serviceapi.ILaunchPluginService.processClassWithFlags(ILaunchPluginService.java:156) ~[modlauncher-10.0.9.jar:10.0.9+10.0.9+main.dcd20f30] {}
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113) ~[securejarhandler-2.1.10.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219) ~[securejarhandler-2.1.10.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229) ~[securejarhandler-2.1.10.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219) ~[securejarhandler-2.1.10.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135) ~[securejarhandler-2.1.10.jar:?] {}
	at java.lang.ClassLoader.loadClass(ClassLoader.java:525) ~[?:?] {}
	at net.minecraft.world.entity.player.Player.<init>(Player.java:144) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.level.ServerPlayer.<init>(ServerPlayer.java:249) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.players.PlayerList.m_215624_(PlayerList.java:422) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading}
	at net.minecraft.server.network.ServerLoginPacketListenerImpl.m_10055_(ServerLoginPacketListenerImpl.java:121) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.network.ServerLoginPacketListenerImpl.m_9933_(ServerLoginPacketListenerImpl.java:70) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.network.Connection.m_129483_(Connection.java:263) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading}
	at net.minecraft.server.network.ServerConnectionListener.m_9721_(ServerConnectionListener.java:142) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading}
	at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:907) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:814) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.client.server.IntegratedServer.m_5705_(IntegratedServer.java:89) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:661) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:251) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at java.lang.Thread.run(Thread.java:833) ~[?:?] {}
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [mixin.ponr.json:FoodDataMixin] from phase [DEFAULT] in config [mixin.ponr.json] FAILED during APPLY
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:636) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinApplyError(MixinProcessor.java:588) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:379) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	... 26 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Critical injection failure: @ModifyConstant annotation on modifyEatMaxFoodLevel could not find any targets matching 'eat(IF)V' in net.minecraft.world.food.FoodData. No refMap loaded. [PREINJECT Applicator Phase -> mixin.ponr.json:FoodDataMixin -> Prepare Injections ->  -> constant$zza000$modifyEatMaxFoodLevel(I)I -> Parse]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.validateTargets(InjectionInfo.java:656) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.findTargets(InjectionInfo.java:587) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.readAnnotation(InjectionInfo.java:330) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.<init>(InjectionInfo.java:316) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.injection.struct.ModifyConstantInjectionInfo.<init>(ModifyConstantInjectionInfo.java:54) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?] {}
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?] {}
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?] {}
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo$InjectorEntry.create(InjectionInfo.java:149) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.parse(InjectionInfo.java:708) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.prepareInjections(MixinTargetContext.java:1311) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.prepareInjections(MixinApplicatorStandard.java:1042) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:393) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:325) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:383) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:365) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	... 26 more
[16:36:24] [Server thread/FATAL] [ne.mi.co.ForgeMod/]: Preparing crash report with UUID 9a8dcfd1-b5f4-43fb-b339-0372561ef97d
[16:36:24] [Server thread/ERROR] [minecraft/MinecraftServer]: This crash report has been saved to: C:\Users\********\AppData\Roaming\ElyPrismLauncher\instances\Test\minecraft\crash-reports\crash-2025-06-17_16.36.24-server.txt
[16:36:24] [Server thread/INFO] [minecraft/MinecraftServer]: Stopping server
[16:36:24] [Server thread/INFO] [minecraft/MinecraftServer]: Saving players
[16:36:24] [Server thread/INFO] [minecraft/MinecraftServer]: Saving worlds
[16:36:24] [Render thread/FATAL] [ne.mi.co.ForgeMod/]: Preparing crash report with UUID 88791a7a-d6e6-4bd1-9063-780d1c4c4180
---- Minecraft Crash Report ----
// My bad.

Time: 2025-06-17 16:36:24
Description: Exception in server tick loop

org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:250) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.service.modlauncher.MixinTransformationHandler.processClass(MixinTransformationHandler.java:131) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.launch.MixinLaunchPluginLegacy.processClass(MixinLaunchPluginLegacy.java:131) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at cpw.mods.modlauncher.serviceapi.ILaunchPluginService.processClassWithFlags(ILaunchPluginService.java:156) ~[modlauncher-10.0.9.jar:10.0.9+10.0.9+main.dcd20f30] {}
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50) ~[modlauncher-10.0.9.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113) ~[securejarhandler-2.1.10.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219) ~[securejarhandler-2.1.10.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229) ~[securejarhandler-2.1.10.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219) ~[securejarhandler-2.1.10.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135) ~[securejarhandler-2.1.10.jar:?] {}
	at java.lang.ClassLoader.loadClass(ClassLoader.java:525) ~[?:?] {}
	at net.minecraft.world.entity.player.Player.<init>(Player.java:144) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.level.ServerPlayer.<init>(ServerPlayer.java:249) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.players.PlayerList.m_215624_(PlayerList.java:422) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading}
	at net.minecraft.server.network.ServerLoginPacketListenerImpl.m_10055_(ServerLoginPacketListenerImpl.java:121) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.network.ServerLoginPacketListenerImpl.m_9933_(ServerLoginPacketListenerImpl.java:70) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.network.Connection.m_129483_(Connection.java:263) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading}
	at net.minecraft.server.network.ServerConnectionListener.m_9721_(ServerConnectionListener.java:142) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading}
	at net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:907) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:814) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.client.server.IntegratedServer.m_5705_(IntegratedServer.java:89) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:661) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:251) ~[client-1.20.1-20230612.114412-srg.jar%23160!/:?] {re:classloading,pl:accesstransformer:B}
	at java.lang.Thread.run(Thread.java:833) ~[?:?] {}
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [mixin.ponr.json:FoodDataMixin] from phase [DEFAULT] in config [mixin.ponr.json] FAILED during APPLY
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:636) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinApplyError(MixinProcessor.java:588) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:379) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	... 26 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Critical injection failure: @ModifyConstant annotation on modifyEatMaxFoodLevel could not find any targets matching 'eat(IF)V' in net.minecraft.world.food.FoodData. No refMap loaded. [PREINJECT Applicator Phase -> mixin.ponr.json:FoodDataMixin -> Prepare Injections ->  -> constant$zza000$modifyEatMaxFoodLevel(I)I -> Parse]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.validateTargets(InjectionInfo.java:656) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.findTargets(InjectionInfo.java:587) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.readAnnotation(InjectionInfo.java:330) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.<init>(InjectionInfo.java:316) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.injection.struct.ModifyConstantInjectionInfo.<init>(ModifyConstantInjectionInfo.java:54) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?] {}
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?] {}
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?] {}
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo$InjectorEntry.create(InjectionInfo.java:149) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.parse(InjectionInfo.java:708) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.prepareInjections(MixinTargetContext.java:1311) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.prepareInjections(MixinApplicatorStandard.java:1042) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:393) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:325) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:383) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:365) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	... 26 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.20.1
	Minecraft Version ID: 1.20.1
	Operating System: Windows 11 (amd64) version 10.0
	Java Version: 17.0.8, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 568584824 bytes (542 MiB) / 1795162112 bytes (1712 MiB) up to 9529458688 bytes (9088 MiB)
	CPUs: 12
	Processor Vendor: GenuineIntel
	Processor Name: 13th Gen Intel(R) Core(TM) i5-1335U
	Identifier: Intel64 Family 6 Model 186 Stepping 3
	Microarchitecture: unknown
	Frequency (GHz): 2.50
	Number of physical packages: 1
	Number of physical CPUs: 10
	Number of logical CPUs: 12
	Graphics card #0 name: Intel(R) Iris(R) Xe Graphics
	Graphics card #0 vendor: Intel Corporation (0x8086)
	Graphics card #0 VRAM (MB): 2048.00
	Graphics card #0 deviceId: 0xa7a1
	Graphics card #0 versionInfo: DriverVersion=32.0.101.5542
	Memory slot #0 capacity (MB): 16384.00
	Memory slot #0 clockSpeed (GHz): 3.20
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 3.20
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 28059.70
	Virtual memory used (MB): 23535.38
	Swap memory total (MB): 3840.00
	Swap memory used (MB): 1716.86
	JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms512m -Xmx9088m
	Server Running: true
	Player Count: 0 / 8; []
	Data Packs: vanilla, mod:tponr, mod:forge, mod:geckolib
	Enabled Feature Flags: minecraft:vanilla
	World Generation: Stable
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'forge'; Server brand changed to 'forge'
	Launched Version: 1.20.1
	ModLauncher: 10.0.9+10.0.9+main.dcd20f30
	ModLauncher launch target: forgeclient
	ModLauncher naming: srg
	ModLauncher services: 
		mixin-0.8.5.jar mixin PLUGINSERVICE 
		eventbus-6.0.5.jar eventbus PLUGINSERVICE 
		fmlloader-1.20.1-47.3.12.jar slf4jfixer PLUGINSERVICE 
		fmlloader-1.20.1-47.3.12.jar object_holder_definalize PLUGINSERVICE 
		fmlloader-1.20.1-47.3.12.jar runtime_enum_extender PLUGINSERVICE 
		fmlloader-1.20.1-47.3.12.jar capability_token_subclass PLUGINSERVICE 
		accesstransformers-8.0.4.jar accesstransformer PLUGINSERVICE 
		fmlloader-1.20.1-47.3.12.jar runtimedistcleaner PLUGINSERVICE 
		modlauncher-10.0.9.jar mixin TRANSFORMATIONSERVICE 
		modlauncher-10.0.9.jar fml TRANSFORMATIONSERVICE 
	FML Language Providers: 
		minecraft@1.0
		lowcodefml@null
		javafml@null
	Mod List: 
		client-1.20.1-20230612.114412-srg.jar             |Minecraft                     |minecraft                     |1.20.1              |DONE      |Manifest: ****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****
		the-point-of-no-return-7116e0e.jar                |The Point Of No Return        |tponr                         |1.1.2               |DONE      |Manifest: NOSIGNATURE
		forge-1.20.1-47.3.12-universal.jar                |Forge                         |forge                         |47.3.12             |DONE      |Manifest: ****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****:****
		geckolib-forge-1.20.1-4.7.1.3.jar                 |GeckoLib 4                    |geckolib                      |**.**.**.**             |DONE      |Manifest: NOSIGNATURE
	Crash Report UUID: 88791a7a-d6e6-4bd1-9063-780d1c4c4180
	FML: 47.3
	Forge: net.minecraftforge:47.3.12
#@!@# Game crashed! Crash report saved to: #@!@# C:\Users\********\AppData\Roaming\ElyPrismLauncher\instances\Test\minecraft\crash-reports\crash-2025-06-17_16.36.24-server.txt
[16:36:26] [Server thread/INFO] [minecraft/MinecraftServer]: Saving chunks for level 'ServerLevel[New World]'/minecraft:overworld
[16:36:26] [Server thread/INFO] [minecraft/MinecraftServer]: Saving chunks for level 'ServerLevel[New World]'/minecraft:the_end
[16:36:26] [Server thread/INFO] [minecraft/MinecraftServer]: Saving chunks for level 'ServerLevel[New World]'/minecraft:the_nether
[16:36:26] [Server thread/INFO] [minecraft/MinecraftServer]: ThreadedAnvilChunkStorage (New World): All chunks are saved
[16:36:26] [Server thread/INFO] [minecraft/MinecraftServer]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[16:36:26] [Server thread/INFO] [minecraft/MinecraftServer]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[16:36:26] [Server thread/INFO] [minecraft/MinecraftServer]: ThreadedAnvilChunkStorage: All dimensions are saved
Process exited with code -1.

```

--- a/src/main/java/org/studio4sv/tponr/mixin/FoodDataMixin.java
+++ b/src/main/java/org/studio4sv/tponr/mixin/FoodDataMixin.java
@@ -21,7 +21,7 @@ public abstract class FoodDataMixin {
         return newFoodLevel;
     }
 
-    @ModifyConstant(method = "eat(IF)V", constant = @Constant(intValue = 20))
+    @ModifyConstant(method = "eat", constant = @Constant(intValue = 20))
     private int modifyEatMaxFoodLevel(int original) {
         return newFoodLevel;
     }

